src/timestamp.cpp

```diff
@@ -0,0 +1,53 @@
+#include "g3log/time.hpp"
+#include "g3log/timestamp.hpp"
+#include <sstream>
+
+
+namespace g3 {
+
+
+	Timestamp::Timestamp(std::string& defaultFormat) :
```

Comment:
Use the posix struct.  On the windows side the winsock or windows.h will provide what is needed.

When you get this closer to finish I can provide the code needed to make it work on Windows